cli.ts

```diff
@@ -44,6 +44,7 @@ async function generate() {
   await Deno.remove("bindings.json");
 }
 
+await Deno.remove("bindings.json");
```

Comment:
Put this in a try catch and ignore the NotFound error (happens on first run)

Reply:
just made the catch a no-op, hopefully shouldnt matter since its only there to fix an error